Tests/NIOSSHTests/Utilities.swift

```diff
@@ -27,3 +29,224 @@ func assertNoThrowWithValue<T>(_ body: @autoclosure () throws -> T, defaultValue
         }
     }
 }
+
+// This algorithm is not secure, used only for testing purposes
+struct TinyEncryptionAlgorithm {
```

Comment:
I'm not sure this is the best idea, but I don't think we can use swift-crypto here as it requires tag authentication, we need some encryption scheme were we can decode just the first block. Do you have any alternative ideas?

Reply:
Yeah, let's just do fixed-byte XOR. Encryption/decryption is XOR with a specific byte. We can then use SHA256 as a MAC. How does that sound?